app-backend/db/src/main/scala/ProjectDao.scala

```diff
@@ -40,7 +40,11 @@ object ProjectDao
 
   def deleteCache(id: UUID) = {
     val result = for {
-      _ <- remove(Project.cacheKey(id))(projectCache, async[IO])
+      _ <- {
+        val cacheKey = Project.cacheKey(id)
+        logger.debug(s"Removing $cacheKey")
```

Comment:
I don't remember, does the logger make this string interpolation lazy if it doesn't print?

Reply:
Yes, it's lazy - it's _ok_ to put in expensive messages according to the [docs](https://github.com/lightbend/scala-logging#scala-logging-)